client/src/components/game/ruler/Ruler.vue

```diff
@@ -160,6 +191,15 @@ export default {
     GameContainer.resetMode()
   },
   methods: {
+    async loadSpecialists () {
+      this.isLoading = true
+      let requests = [ SpecialistService.getCarrierSpecialists( this.$store.state.game._id ) ]
+
+      const responses = await Promise.all(requests)
+
+      this.carrierSpecialists = responses[0].data
```

Comment:
I believe the list of carrier specs already exists in the state. You shouldn't need to call the API for this.

Check the hire carrier specialist component.